lib/rucio/core/did.py

```diff
@@ -43,12 +43,13 @@
 
 if TYPE_CHECKING:
     from collections.abc import Callable, Sequence
-    from typing import Any, Optional, Union
+    from typing import Any, Dict, Optional, Union, List
     from sqlalchemy.orm import Session
     from sqlalchemy.schema import Table
     from rucio.common.types import InternalAccount, InternalScope
 
     LoggerFunction = Callable[..., Any]
```

Comment:
```suggestion
    from rucio.common.types import InternalAccount, InternalScope, LoggerFunction
```

`LoggerFunction` is defined in `types.py`: https://github.com/rucio/rucio/blob/master/lib/rucio/common/types.py#L107C1-L107C15